src/libs/actions/Report.js

```diff
@@ -1398,6 +1398,33 @@ function handleInaccessibleReport() {
     navigateToConciergeChat();
 }
 
+/**
+ * Creates a policy room and fetches it
+ * @param {String} policyID
+ * @param {String} reportName
+ * @param {String} visibility
+ * @return {Promise}
+ */
+function createPolicyRoom(policyID, reportName, visibility) {
+    return API.CreatePolicyRoom({policyID, reportName, visibility})
+        .then((response) => {
+            if (!response) {
+                return;
+            }
+            if (response.jsonCode !== 200) {
+                console.error(response.message);
+                return;
+            }
```

Comment:
Is a missing response ever going to occur? I think we can just make this and remove the !response

if (response && response.jsonCode !== 200)